clang/test/Driver/flang/dev_null_as_input.f95

```diff
@@ -0,0 +1,10 @@
+! REQUIRES: classic_flang
+
+! Check that the driver invokes flang1 correctly for preprocessed fixed-form
+! Fortran code.
+
+! RUN: %clang --driver-mode=flang -E -target x86_64-unknown-linux-gnu -c -</dev/null -### 2>&1 \
```

Comment:
this is OK, although it is hard to read. Could you put one space between `-` and `<`? Namely `-c - </dev/null`